src/UserSwitch/UserSwitch.tsx

```diff
@@ -35,7 +34,7 @@ const StyledWrapper = styled(Box)(() => ({
 }));
 
 interface Props {
-  Actions?: JSX.Element | JSX.Element[];
+  Actions?: JSX.Element | (JSX.Element | null)[];
```

Comment:
I don't think there is a need for this change. The array should not contain `null` values.